store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreSchemaTest.java

```diff
@@ -117,10 +115,7 @@ public void shouldBeAbleToAddGraphsWithSchemaCollisions() throws OperationExcept
         Collection<String> graphIds = fStore.getAllGraphIds(testUser);
 
         // Then
-        HashSet<String> expected = new HashSet<>();
-        expected.addAll(Arrays.asList("a", "b", "c"));
-
-        assertEquals(expected, graphIds);
+        assertThat(graphIds).containsExactlyInAnyOrder("a", "b", "c");
```

Comment:
I think order did matter.

Reply:
HashSets are not ordered so order wasn't checked before and `graphIds` is returned in a different order every time as it is also made from a Set

Reply:
nice one.